apps/triageparty-release/deployment.yaml

```diff
@@ -4,16 +4,16 @@ metadata:
   name: triage-party
   namespace: triageparty-release
   labels:
-    app: triage-party
+    app: triageparty-release
```

Comment:
I would have preferred that we keep the current labels and add new one referring the community group:

```yaml
labels:
	app: triage-party
	group: sig-release
```

@spiffxp WDYT ?

Reply:
I was looking for a term we can use to enforce same-name conventions for:
1. a namespace on aaa
2. a directory under apps/ that holds (or generates) all k8s resources in that namespace
3. an rbac group that is authorized to deploy k8s resources to that namespace
4. (maybe overkill?) a label on all resources deployed from the aforementioned apps subdirectory (vs. resources dynamically created by the k8s resources or its users) <-- this is why the `app:` value is being renamed

I think I latched onto "app" because it's what I feel like we've been calling them, but it's actually mixing two discrete meanings, since they've been the same for many of the, uh, "things" we're running in aaa.

To borrow from OO terminology, I think your objection is that the "app:" label better fits the "class" and not the "instance" of app.  In Helm parlance "app" would be the 1 Chart, not the N Releases.  In kpt terms, I think "app" would be the 1 Package, not the N ResourceGroups.  

Since naming is hard I think we should iterate on how to resolve this on a PR against https://github.com/kubernetes/k8s.io/blob/main/apps/README.md

In the interest of unblocking, does the label change break things now?  If it does, let's revert.  If it doesn't, are you OK if we iterate on this post-merge?

Reply:
I know the question was directed at @ameukam - but I'm thinking I would be most comfortable reverting the `app:` label back to the original, and waiting to figure out the "story" before making a change. We can always come back and update.

Reply:
+1 for to figure out this later.